app/src/main/java/com/github/siela1915/bootcamp/ShoppingListViewHolder.java

```diff
@@ -1,15 +1,38 @@
 package com.github.siela1915.bootcamp;
 
 import android.view.View;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
 import android.widget.TextView;
 
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
 
-public class ShoppingListViewHolder extends RecyclerView.ViewHolder{
-    TextView shopping_items;
-    public ShoppingListViewHolder(@NonNull View itemView) {
+import java.util.List;
+
+public class ShoppingListViewHolder extends RecyclerView.ViewHolder implements CompoundButton.OnCheckedChangeListener {
+    CheckBox checkBox;
```

Comment:
Same remark, better to have private class attributs and define setters and getters if needed